### PR TITLE
Release/0.11

### PR DESCRIPTION
**Pull request by anxolin**
`git log master..release/0.11 --oneline --decorate=0`

5430079 Fix button layout desktop/mobile. (#889)
188c353 Support dark mode in the orderbook. (#895)
75501ea Misc style fixes (#896)
a931d5a Fix button clicks (#894)
f12e889 Volume: Leave only accumulated volume in the tooltip (#893)
b830583 Update infura key and set some env vars (#890)
4da004f Update README.md (#891)
bf15631 fixes saved trade params & second tx error (#886)
c32b4eb Add liquidity message. (#885)
b31de36 Update dune dashboard (#888)
9e8cdf2 Better token add modal from URL (#866)
3156321 Order book: Improve logic of which tokens are selected (#861)
4afb9a4 Show network name (#872)
3dcddfd Remove tooltip in trade widget (#883)
c6c8b60 Reduce length of FAQ.md (#783)
313c8d1 Fix 875 token address on rinkeby (#878)
c94e537 790/fetch prices from dex price estimator (#834)
7fc9ed0 Update dex-js (#865)
64da5ea Add icon to make the order book more visible (#868)
71478b8 Not reversing orders array in place because that's bad (#870)
aabbbde Add link to deposit to encourage to do the deposits (#857)
bfa3bb0 Css fixes pre launch (#863)
f3e6e25 Fix wrong addresses on rinkeby (#858)
007560c Add balances message. (#862)
48bfa64 Deposit message (#860)
860edfd Order book btn (#850)
bc36180 Add token in TokenSelector (#780)
4d19206 Reversing orders sorting. Latest on top (#853)
819f07d Incorporated changes after Chris' Review (#785)
a91f980 837/remove dfusion (#842)
0a8f978 Refactor default network id logics (#849)
9aa45b1 Wrap ether refactor (#840)
c1b17cd Update About.tsx (#843)
9ee8a67 Orderbook/initial design (#835)
4eeb287 [Orderbook] Rendering the transitive orderbook (#813)
d919482 Fix path for to the fileMock in the tests (#838)
26c944c 695/add schema validation trade page (#782)
a1f5b50 redirect to https (#829)
1f7b52c 655/eternal darkness (#792)
ff8d4dd 795/preselect liquidity tokens (#832)
abd9e4e Add order book page (#800)
daebbaa Dynamically loading tokens ids, per network, first time getTokens is called (#824)
e8c4cc3 Changed answer format (#799)
4504e9a Orders widget - keep order tabs visible (#826)
9d4f30e Tooltip text change (#786)
5b8d1c2 Use yarn instead of npm (#807)
1c0dc87 Update URL dev portal (#811)
689f2b4 Bump @babel/core from 7.8.7 to 7.9.0
dc586dc Bump @types/styled-components from 4.4.3 to 5.0.1
4b02128 Bump @babel/register from 7.8.6 to 7.9.0
2f3d6d5 Bump eslint-plugin-import from 2.20.1 to 2.20.2
3d21ce5 Bump jest from 25.1.0 to 25.2.4
0c8d834 Merging both lists of tokens to avoid duplicates (#793)
f832aa9 Merge hotfix 0.10.1 to develop (#796)
afc7568 758/unreliable enable token (#768)
05034f4 Add any token (#760)
0d7c48a Merge release 0.10 into develop (#781)
d05ffa7 Removing unused import
b8dac53 Avoid displaying receive amount as NaN (#761)
feb6e96 Reseting prices when switching tokens even if estimation is null (#762)
e2fb104 Trade page global state (#747)
2fe6a09 Adapting to TheGraph model changes (#754)
abacc05 Updating validation to not prevent default in case input is not valid (#753)
40b298e Update FAQ.md (#756)
b2eac3d Add tooltips text (#731)
0b8b38b Select with arrows (#748)
7c34703 use last balance on error (#751)
03b37e8 658/use price estimation (#745)
4d62452 TheGraphApi and getPriceEstimation service (#733)
dd30c65 Tokens search+hide (#725)
f77ad5f more robust noOptionsMessage check (#739)
53723bf 714/selector styling (#736)
74a2081 Remove todo and test (#740)
eeba2a1 Restore commits (#737)
6c348e5 tabIndex order in \/trade (#732)
b697950 change widget size on/connect-wallet (#728)
83d99a7 FAQ and text changes (#730)
9fe8652 Yet another fix for token selector (#727)
fe3fc01 Merge pull request #724 from gnosis/remove_datatype
23b20ad 691/use max locale (#721)
4f3fb73 Fixing filled amount for pending orders (#722)
2deb055 remove datatype
3b28d81 style based on readonly attribute
c2ed8da Merge pull request #720 from gnosis/connect_wallet_action
70571f6 fix lint:fix script
4515915 add link
18078cb style on hover
fda7788 allow to connect by clicking wallet img
43f65b4 696/trade page formatting (#711)
b7af0d2 Pooling interface rework (#683)
47f1130 Merge pull request #715 from gnosis/md_FAQ
8acc555 Merge pull request #689 from gnosis/onChain_analytics
09e560e OnChain analytics screen size (#703)
64b1a3b Add links to devportal in liquidity page (#709)
02ba72b use attribute plugin for markdown links
a994245 rename Component
0000d8a Rotate OrdersPage chevron on click (#713)
9300759 fix names
697fbef add link to \/faq
f7fb9ec add \/faq route
4078ddc add FAQ page
f8c41d4 add FAQ.md asset
26fa2d5 add markdown loader
709bcff add MIN_GAS_PRICE substitution when possible to fit flags
2ce407d more renames
1b95cb4 smaller flagWithMaps destructure
0ef72b4 add comments
4ac1491 rename stuff
3b01090 Fix trade errors (#712)
6f4eb6e Added custom react-select component for new styling (#626)
13ab93f 500/Fix NaN prices on orders page (#706)
18a46b7 694/make buttons more disably (#704)
baf9887 Feature/misc style fixes (#656)
a22b3fa change styles not to use 100vh (#705)
10641c2 fix comment
db65e74 update userPrint on screenSize change
6c97936 add screenSize to userPrint
f67025e add screenSize flags
56c4c95 add media query detection
a60e29d Updating dex-js to add Chai
ac469f8 Misc changes (#698)
2b20248 Bump typescript from 3.7.5 to 3.8.3 (#669)
a5200ab remove log
09572f7 fix typo
ce9c0d1 add detect-browser dependency
826638f fix WalletApiMock types
54f2247 add userPrint generation to WalletApi
7884f65 add our specific encoder/decoder
fc226ed export utils/flagCodes
b84097d add tests for earmmarking function
2f19d32 add earmarking function
8cd7903 add decoder tests
8d0758d add encoder tests
47157b5 add decoder
6a0a57b add encoder
e483d60 add tentative flags
ea238da Refactoring pending and regular orders classification on orders page
15ea5b8 Bump eslint-plugin-react from 7.18.0 to 7.18.3
44ed383 Bump @types/enzyme-adapter-react-16 from 1.0.5 to 1.0.6
ab8bec3 Bump @gnosis.pm/dex-js from 0.1.12-RC.0 to 0.1.12
fb942e5 Bump react-dev-utils from 10.0.0 to 10.2.0
3d41cc9 Merge branch 'master' into merge-hotfix-v0.9.1
0f00636 autofocus deposit field (#682)
1a2ca11 Non-blocking Deposit/Withdraw modals (#677)
d7b0d42 Display balance on trade page token selector (#599)
67f578c 0.9.0 master merged into develop (#671)
86b8cd5 Visible button fix (#663)
31e0044 Typo fix: bach -> batch
030e606 Bump fork-ts-checker-webpack-plugin from 4.0.3 to 4.0.4
b830f7b Triggering wallet connection from trade page on submission (#598)
2bcf45a hack for modali screen flash (#650)
8c1ec27 Render optimization (#647)
61f4c1b Do not disable the send button (#642)
c725805 Close form on Escape (#637)
eb336a5 527/set order start time (#608)
25a3dbf Fix price issue (#631)
09f4ae6 Delete unused column (#641)
1d1a5c7 Autofocus sell input (#638)
4ea8542 Overall layout updates. (#635)
2f6d6e5 Bump @walletconnect/web3-provider from 1.0.0-beta.46 to 1.0.0-beta.47
b4980d9 Bump @types/jest from 25.1.1 to 25.1.3
2692ece Fix balance always on error state (#632)
2ebc209 Hide todo (#624)
3fd9473 Misc style updates. (#607)
737b5b9 fix button.type
50eb8e1 Hide functionalities not included in ethcc version (#617)
17d6cfb Hide deposit and enable for receive token (#609)
4df3344 Improved connect wallet in orderspanel. (#614)
1123dfb Fix form submission (#616)
4bbe470 Feature/toast and fonts design (#601)
4823813 Merge branch 'master' into develop
2fdf9a7 correct deps for readOnly register inputRef (#605)
54a617c 481/add price (#551)
8617685 Improve tokenSelector styles. (#597)
9f9634d Improve logs (#576)
1592733 Balances overview styles update. (#586)
4393f61 528/show from date on orders (#584)
30ae106 Fixing orders not loading on landing page first load
2c2e6b7 588/wallet not connected orders page (#590)
f515e2c Bump eslint-plugin-jest from 23.6.0 to 23.8.0
cd8df63 Feature/orders panel design (#582)
f8e4bc3 Added favicon. (#585)
367297b smart disclaimer - persist close (#577)
fa29524 Feature/develop design progress (#565)
b3b2bf4 Child element must be a React component
09e8a2e Remove some debug logs (#573)
6a31aa1 483/liquidity orders tab (#571)
e41a7fb Merge branch 'master' into develop
217dc17 Tooltips sample integration (#566)
9c73258 Added total amount to orders page (#564)
2b60863 Merge branch 'master' into develop
483da77 Addessing pending comments from PR https://github.com/gnosis/dex-react/pull/553#discussion_r380239086
f1ca63f Merge remote-tracking branch 'origin' into develop
3fb4cec Merge redesign onto develop (#553)
05e7e79 Merge remote-tracking branch 'origin/master' into develop
a1c402a Bump @types/enzyme from 3.10.4 to 3.10.5
53c7e21 Bump @types/styled-components from 4.4.2 to 4.4.3
8d7d6a1 Bump @types/node from 13.7.0 to 13.7.1
972c8de Bump @fortawesome/free-solid-svg-icons from 5.12.0 to 5.12.1
9d240f2 Bump @typescript-eslint/parser from 2.18.0 to 2.19.2
b525a45 Fixing bad merge (by me 😅)
b45b839 Refactor format utils and constants (#501)
e0bc970 482/define validity date (#509)
49939d9 477/ notifications displayed longer (#524)
d7ec036 455/global orders (#520)
19f907c select input text on focus (#518)
c29283c Lazy loading + prefetching (#502)
ffbbb30 start with base tag (#522)
50abfbb When a token is not registered in the exchange, do not allow the order to be placed
662eb5c fix variable (#516)
b902aba 478.5/optimistic liquidity (#512)
2c40603 check if new state is needed (#513)
01056db 478/optimistic orders (#498)
bbf658d useOrders v2 (#510)
7ac8826 Bump @fortawesome/free-regular-svg-icons from 5.12.0 to 5.12.1
8b5fcd2 Bump @typescript-eslint/eslint-plugin from 2.18.0 to 2.19.0
bf54dcc Bump rimraf from 3.0.1 to 3.0.2
8c8c328 Bump @types/node from 13.5.2 to 13.7.0
c47828a Bump @fortawesome/fontawesome-svg-core from 1.2.26 to 1.2.27
2dcd92a Bump enzyme-to-json from 3.4.3 to 3.4.4
d490ff6 Bump react-hook-form from 4.7.2 to 4.8.0
ef59150 Bump webpack-dev-server from 3.10.1 to 3.10.2
17ee9de Bump web3 from 1.2.5 to 1.2.6
13b1946 Merge branch 'master' into develop
d95d242 455/paginating orders page (#489)

### CODE REVIEW

**Comment by Velenir** [comment]
I'd like to see #898 included, but not urgent

And , oh my the conflicts!!

File: src/hooks/useAddTokenModal.tsx [hunk 1]
```diff
@@ -0,0 +1,283 @@
+import React, { useState, useCallback, useRef, useEffect } from 'react'
```
Comment (alfetopito) [inline src/hooks/useAddTokenModal.tsx:1]: This hook doesn't seem to be used, and maybe it has been replaced by `src/hooks/useBetterAddTokenModal.tsx`

Remove?

**Comment by gnosis-info** [comment]
Travis automatic deployment:
 https://pr897--dexreact.review.gnosisdev.com

File: src/hooks/useAddTokenModal.tsx (hunk 1 above)
Comment (Velenir) [reply src/hooks/useAddTokenModal.tsx:1]: Oops, yes you are exactly right. I forgot about it. Please remove

File: src/App.tsx [hunk 2]
```diff
@@ -85,7 +92,8 @@ const App: React.FC = () => (
             <PrivateRoute path="/liquidity" exact component={Strategies} />
             <PrivateRoute path="/wallet" exact component={Wallet} />
             <Route path="/about" exact component={About} />
-            <Route path="/source-code" exact component={SourceCode} />
+            <Route path="/faq" exact component={FAQ} />
+            <Route path="/book" exact component={OrderBook} />
```
[the remark this answers is not in the document]
Comment (anxolin) [reply src/App.tsx:96]: Probably not, but If we do sth, better in develop. 

I think it's not bad to leave it, actually we can leave this page as an easter egg, and add the hops params. @fleupold would be happy if we do. 
The page is lazy loaded, so I think it doesn't add anything to the bundle

File: src/api/dexPriceEstimator/DexPriceEstimatorApi.ts [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+}
+
+export interface Params {
+  networkIds: number[]
+}
+
+function getDexPriceEstimatorUrl(networkId: number): string {
+  const networkName = getNetworkFromId(networkId).toLowerCase()
+  // TODO: use prod endpoint when available https://github.com/gnosis/dex-react/issues/869
+  return `https://price-estimate-${networkName}.dev.gnosisdev.com/api/v1/`
```
[the remark this answers is not in the document]
Comment (anxolin) [reply src/api/dexPriceEstimator/DexPriceEstimatorApi.ts:42]: We have a issue for that. This PR is for a Release candidate, let's do that in develop or a release branch

File: src/assets/md/FAQ.md [hunk 4]
```diff
@@ -0,0 +1,33 @@
+**What is Mesa?**
+
+Mesa is a decentralized application built on the **Gnosis Protocol**, a decentralized trading protocol for ERC-20s. Instead of using one order book per token pair, the Gnosis Protocol relies on batched auctions and a multidimensional orderbook. These mechanisms enable ring trades, in which liquidity amongst all assets tradable on the protocol is shared. Read more about the protocol’s mechanism on the Gnosis [DevPortal](https://docs.gnosis.io/protocol/docs/introduction1/).
```
[the remark this answers is not in the document]
Comment (anxolin) [reply src/assets/md/FAQ.md:3]: Mmm you are right! something not good happen in one merge. I'll review this. Good catch

[the remark this answers is not in the document]
Comment (anxolin) [reply src/assets/md/FAQ.md:3]: Okay, that's not merged, still pending for Kei review https://github.com/gnosis/dex-react/pull/892/files

You scared me, I thought something really wrong happened in the merge :) 
Great call, and good eye though!

[the remark this answers is not in the document]
Comment (alfetopito) [reply src/assets/md/FAQ.md:3]: Ah, still in draft https://github.com/gnosis/dex-react/pull/892

File: src/hooks/useAddTokenModal.tsx (hunk 1 above)
Comment (alfetopito) [reply src/hooks/useAddTokenModal.tsx:1]: @Velenir

Comment (alfetopito) [reply src/hooks/useAddTokenModal.tsx:1]: I'll do it.

Comment (anxolin) [reply src/hooks/useAddTokenModal.tsx:1]: Still used in develop, for example in `TokenOptionItem` but probably YES, although this is not the merge to do it. We should create a branch and merge it to develop

**Pull request merged**